main.js

```diff
@@ -1735,6 +1761,7 @@ addStealExtension(function addCacheBust(loader) {
 	System.ext = Object.create(null);
 	System.logLevel = 0;
 	System.forceES5 = true;
+	System.transpileAllFormats = true;
```

Comment:
Not sure if this is the right option. In the PR we said `transpile` would be it.

Also this has to default to false or it will be a breaking change.

Reply:
`steal.loader.transpile` is a function that's used to do transpilation. We could support the `transpile: true` option, but this would be tricky to implement. I don't think a slightly better name is actually worth it and `transpileAllFormats` is just fine.